apps/indexer/lib/indexer/celo/metrics_cron.ex

```diff
@@ -144,6 +145,14 @@ defmodule Indexer.Celo.MetricsCron do
     :telemetry.execute([:indexer, :blocks, :last_block_number], %{value: last_block_number})
   end
 
+  def nft_token_instances_metrics do
+    unfetched_erc_721_token_instances_count = Chain.unfetched_erc_721_token_instances_count()
+
+    :telemetry.execute([:indexer, :nft, :unfetched_erc_721_token_instances], %{
```

Comment:
It's fine to do this manually as you can explicitly set the event name, but just checking you are aware of the telemetry wrapper at `apps/explorer/lib/explorer/celo/telemetry/telemetry.ex` which wraps common event emission